src/app/clusters/ota-requestor/GenericOTARequestorDriver.cpp

```diff
@@ -201,7 +207,7 @@ void GenericOTARequestorDriver::UpdateNotFound(UpdateNotFoundReason reason, Syst
     else
     {
         ChipLogProgress(SoftwareUpdate, "UpdateNotFound, not scheduling further retries");
-        StartDefaultProviderTimer();
+        StartPeriodicQueryTimer();
```

Comment:
Why not do `StartSelectedTimer(SelectedTimer::kPeriodicQueryTimer);` here?

Reply:
This just got resolved with no explanation?

Reply:
I made the change so I resolved it.  Looks like the change didn't make it into the branch again for some reason.  Let try to make the change again.